Path for kubectl hardcoded

### Code of Conduct

- [X] I have read and agree to the [Code of Conduct](https://github.com/vmware/power-validated-solutions-for-cloud-foundation/blob/main/CODE_OF_CONDUCT.md).
- [X] Vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue initial description to help the maintainers prioritize.
- [X] Do not leave "+1" or other comments that do not add relevant information or questions.
- [X] If you are interested in working on this issue or have submitted a pull request, please leave a comment.

### VMware Cloud Foundation

5.2

### Module Version

2.11.0.1057

### PowerShell Version

7.4.4

### PowerCLI Version

13.3

### PowerVCF Version

2.4.1.1000

### Guest Operating System

Windows Server 2019

### Environment Details

_No response_

### Description

The script expects kubectl to be installed in C:\kubectl\bin.  Please consider changing this look in %PATH% on Windows and $PATH on Linux/MacOS. 

### Error or Debug Output

 [ERROR] Unable to find path to kubectl on the local machine: PRE_VALIDATION_FAILED"

### Expected Behavior

Looks for the executable in the system path 

### Actual Behavior

Looks in hardcoded path

### Steps to Reproduce

Do not install kubectl in C:\Kubectl\bin, but rather place in a system path.

### Log Fragments and Files

_No response_

### Screenshots

_No response_

### References

_No response_

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -4,8 +4,10 @@
 
 > Release Date: Not Released
 
-- Enhanced `Invoke-InvDeployment` cmdlet to remove the need for Workspace ONE Acceess and add in the `admin` password to the VMware Aria Suite Lifecycke password locker.
+- Enhanced `Invoke-InvDeployment` cmdlet to remove the need for Workspace ONE Access and add in the `admin` password to the VMware Aria Suite Lifecycle password locker.
 - Enhanced `Get-LocalAccountLockout` cmdlet to handle "null" values got as "Not Configured" when appliance is not configured.
+- Enhanced `Export-DriJsonSpec` cmdlet to define the path to the Kubectl utility.
+- Enhanced `Start-DriMenu` cmdlet to use the new JSON value for the path to the Kubectl utility.
 
 ## v2.11.0
 

--- a/PowerValidatedSolutions.psd1
+++ b/PowerValidatedSolutions.psd1
@@ -11,7 +11,7 @@
     RootModule = 'PowerValidatedSolutions.psm1'
 
     # Version number of this module.
-    ModuleVersion     = '2.11.1.1001'
+    ModuleVersion = '2.11.1.1002'
 
     # ID used to uniquely identify this module
     GUID              = 'a6dfed7b-65d2-4da2-bdcc-7f3d3df9b75d'

--- a/PowerValidatedSolutions.psm1
+++ b/PowerValidatedSolutions.psm1
@@ -9682,6 +9682,7 @@ Function Export-DriJsonSpec {
                 'namespaceViewUserGroup'        = $pnpWorkbook.Workbook.Names["group_gg_kub_readonly"].Value
                 'tanzuNamespaceName'            = $pnpWorkbook.Workbook.Names["k8s_cluster_name"].Value
                 'vmClass'                       = $pnpWorkbook.Workbook.Names["k8s_vm_class"].Value
+                'kubectlPath'                   = "C:\Kubectl\bin"
             }
             Close-ExcelPackage $pnpWorkbook -NoSave -ErrorAction SilentlyContinue
             $jsonObject | ConvertTo-Json -Depth 12 | Out-File -Encoding UTF8 -FilePath $jsonFile
@@ -60193,12 +60194,14 @@ Function Start-DriMenu {
                 }
                 5 {
                     if (!$headlessPassed) { Clear-Host }; Write-Host `n " $submenuTitle : $menuItem05" -Foregroundcolor Cyan; Write-Host ''
-                    Invoke-DriDeployment -jsonFile ($jsonPath + $jsonSpecFile) -certificates $certificatePath -binaries $binaryPath -kubectlPath "C:\Kubectl\bin\"
+                    $jsonData = (Get-Content -Path ($jsonPath + $jsonSpecFile)) | ConvertFrom-Json
+                    Invoke-DriDeployment -jsonFile ($jsonPath + $jsonSpecFile) -certificates $certificatePath -binaries $binaryPath -kubectlPath $jsonData.kubectlPath
                     waitKey
                 }
                 6 {
                     if (!$headlessPassed) { Clear-Host }; Write-Host `n " $submenuTitle : $menuItem06" -Foregroundcolor Cyan; Write-Host ''
-                    Invoke-UndoDriDeployment -jsonFile ($jsonPath + $jsonSpecFile) -binaries $binaryPath -kubectlPath "C:\Kubectl\bin\"
+                    $jsonData = (Get-Content -Path ($jsonPath + $jsonSpecFile)) | ConvertFrom-Json
+                    Invoke-UndoDriDeployment -jsonFile ($jsonPath + $jsonSpecFile) -binaries $binaryPath -kubectlPath $jsonData.kubectlPath
                     waitKey
                 }
                 B {

